buildSrc/src/main/kotlin/splunk.java-conventions.gradle.kts

```diff
@@ -5,6 +5,11 @@ plugins {
 repositories {
   mavenCentral()
   maven {
+    name = "sonatypeReleases"
+    url = uri("https://oss.sonatype.org/content/repositories/releases/")
```

Comment:
I added the sonatype releases repo here, to avoid waiting for maven central to get the latest `signalfx-metrics` release